gov2/cloudwatch/README.md

```diff
@@ -1,108 +1,54 @@
-# AWS SDK for Go V2 code examples for Amazon CloudWatch
+# Amazon CloudWatch examples for the AWS SDK for Go (V2)
```

Comment:
Short name for service and SDK in title.

Reply:
Changed to: CloudWatch examples for the SDK for Go (V2)